website/app/GeoKrety/HealthState.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace GeoKrety;
+
+class HealthState {
+    const STATE_OK = 'ok';
+    const STATE_KO = 'ko';
+    const VALID_STATES = [self::STATE_OK, self::STATE_KO];
+
+    public $state = self::STATE_OK;
```

Comment:
What about a default to `null`? This may inform that the check has not been run at all.

Reply:
Oo just missed that one wait

Reply:
done